matlab/tdcr/draw_tdcr.m

```diff
@@ -78,9 +78,9 @@
     
     %% Projections
     if options.projections
-        plot3(g(:,13), ax.YLim(1)*ones(length(g)), g(:,15), 'LineWidth', 2, 'Color',[0 1 0]); % project in x-z axis
```

Comment:
There was a bug in the TDCR projections that led to the one segment TDCR looking weird when projections are enabled.

Before Fix:
![Figure_2](https://github.com/ContinuumRoboticsLab/CRVisToolkit/assets/46078134/a3d6deb9-43d5-463f-a013-6bf311b8daab)

After Fix:
![Figure_9](https://github.com/ContinuumRoboticsLab/CRVisToolkit/assets/46078134/06a37997-b85f-41ef-aed6-a09db3e60940)